chunker/application.py

```diff
@@ -0,0 +1,61 @@
+from util import log
+
+import pipeline, tiler
+import re, os, sys, time
+import boto.sqs
+
+def parse_filename(path):
+    name = os.path.splitext(os.path.basename(path))[0]
+    m = re.match("""(?P<datatype>[^_]+)_amon_BCSD_(?P<context>[^_]+)_r1i1p1_CONUS_(?P<model>[^_]+)\_""", name)
+    if m:
+        datatype = m.group("datatype")
+        context = m.group("context")
+        model = m.group("model")
+        return (datatype, context, model)
+
+    m = re.match("""(?P<datatype>[^_]+)_quartile75_amon_(?P<context>[^_]+)_CONUS_.*""", name)
+    if m:
+        datatype = m.group("datatype")
+        context = m.group("context")
+        model = "ensemble"
+        return (datatype, context, model)
+
+    log("Could not parse name " + name)
+    return None
+
+if __name__ == '__main__':
+    q_name = os.environ['CHUNKER_QUEUE']
+    target_bucket = os.environ['CHUNKER_TARGET_BUCKET']
+
+    log("KEY: " + os.environ['AWS_ACCESS_KEY_ID'])
```

Comment:
Use logging.getLogger instead of this custom log function (which doesn't exist anymore)